docs/integrations/sources/paypal-transaction.md

```diff
@@ -260,7 +260,7 @@ ___
 * **Number of days per request:**  The maximum supported date range is 31 days.
 * **Historical Data:**  You can't retrieve more than 3yrs of data for the `transactions` stream. For `dispute_start_date` you can only retrieve 180 days of data (see specifications per stream) 
 * `records_per_request`: The maximum number of records in a single request are 10K (API Server restriction)
-* `page_size`: The maximum page size is 500. This has been configured by default.
+* `page_size`: The maximum page size differs in every stream. Thr Max allowed pages has been configured by default.
```

Comment:
```suggestion
* `page_size`: The maximum page size differs in every stream. The Max allowed pages has been configured by default.
```

Reply:
I don't quite understand what "the max allowed pages" is in this context, and where exactly it's configured. Can you point me to where I can learn more? Did you mean maximum page size?

Reply:
The Maximum page size (items) per page for every stream is different. So, one stream has a page size of 20, another of 100m and so on. That is why I clarify it here, but every section has the current max configured page_size. You can check each stream section in the already publish documentation (or in this code) in the tables, or directly in paypal API documentation.

Reply:
Let me clarify my question.

> The maximum page size differs in every stream. The Max allowed pages has been configured by default.

There are two different things in this paragraph: 

1. **Maximum allowed page size: you're saying it differs per stream.** My asks:
   - let's clarify that the specific `page_size` values are described below for each stream.
   - In some streams, for example [Transactions](https://github.com/airbytehq/airbyte/blob/8086648bd067e1e2118a4f6d411f01db104d4dec/docs/integrations/sources/paypal-transaction.md#transactions-stream), the `page_size` is described as `Max 500 (F)`. Can we separate configurable parameters and constants? If there's a constant value, why include "Max" with it?
2. **The Max allowed pages has been configured by default.** I think what you mean is that you've already pre-configured maximum allowed `page_size` for each stream. Not _maximum allowed number of pages_, but _maximum allowed number of items per page_. I didn't understand the sentence initially. Would you be open to reword it to "Each stream's configuration is already set to request the highest `per_page` available in the API"?

Reply:
1. I don't really understand the ask. The table contains MAX and below the description when you find an F (F: Fixed Value. This means it is not configurable.)

2. You are correct, I made a syntax mistake, it should have been "the maximum allowed number of items per page."